conda-forge.yml

```diff
@@ -1,8 +1,6 @@
 azure:
   store_build_artifacts: true
 bot:
-  abi_migration_branches:
-  - 3.15
```

Comment:
@ryanvolz, is 3.15 still in use anywhere? It seems severely out of date (the release is almost [4 years](https://github.com/EttusResearch/uhd/releases/tag/v3.15.0.0) old; the abi-branch got [added](https://github.com/conda-forge/uhd-feedstock/commit/d82b5bfa47a7ae8eab17ef0971a5b358ad161a96) 2.5 years ago), and I could not [find](https://github.com/search?q=org%3Aconda-forge%20uhd%203.15&type=code) mentions anywhere in conda-forge. I'm pretty sure it's not used anymore?

Reply:
I just dropped support for `gnuradio 3.8` a few days ago, and that was the only thing still using `uhd 3.15`. So yes, it makes sense to drop the 3.15 branch from support here.